.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ defaults:
         username: $DOCKER_CLOUD_USER
         password: $DOCKER_CLOUD_PASSWORD
       environment:
-        N_DAYS: 3
+        N_DAYS: 7
```

Comment:
Can we change this back? It was dropped to 3 because:

- It allows us to check the env var is getting passed on correctly (7 is the default when you spin up)
- It reduces the time to run the tests

Reply:
I don't think the test runtime is noticeably affected (since we still create only a relatively small amount of rows per day), but checking that the env var is passed in is a good point. I'll change it back.